google_speech/__init__.py

```diff
@@ -269,6 +295,11 @@ def cl_main():
                           default="normal",
                           dest="verbosity",
                           help="Level of logging output")
+  arg_parser.add_argument("-o",
+                          "--output",
+                          default=None,
+                          dest="output",
+                          help="Name of the output file")
```

Comment:
This options completely changes the behavior of the program, so for the description I would rather like something like "Outputs audio data to this file instead of playing it" to clearly states that nothing will be played when the option is used.

Also I think it would be a good idea to add something like:
```
if (args.output is not None) and args.sox_effects:
  logging.getLogger().debug("Effects are not supported when saving to a file")
  exit(1)
```
to clearly warn the users that effects and saving to file are not compatible.